<launch>
<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/wheelchair_robot/left_camera/image_raw" />
</node>
</launch>
